rootfs/etc/pulse/system.pa

```diff
@@ -45,3 +50,14 @@ load-module module-always-sink
 
 ### Enable positioned event sounds
 load-module module-position-event-sounds
+.endif
+
+# Load additional user configuration, if present                             
+#                                                                                                              
+# Using nofail has two effects here:                                           
+#  1) Causes PulseAudio to log a message at startup if the user configuration  
+#     is not present to increase visible of the corresponding configuration                                    
```

Comment:
By that you mean so users learn about the possibility?

Seems weird to me to do that through an error 😅 . I'd rather go with ifexists here, and document the capabilities somewhere.